person.rb

```diff
@@ -2,20 +2,19 @@ class Person
   reader_attr :id
   accessor_attr :name, :age
 
-  def initialize(id, age, name = 'Unknown', parent_permission: true)
-    @id = id
+  def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
   end
```

Comment:
- [ ] (OPTIONAL) This is not really optional. The `id` should be added as an instance variable under `@parent_permission` and it would be great if each Person `id` could be unique. You can accomplish this easily by setting `@id = Random.rand(1..1000)`

Reply:
thank you, ill implement that